pagerduty/resource_pagerduty_service_test.go

```diff
@@ -1093,6 +1121,47 @@ resource "pagerduty_service" "foo" {
 `, username, email, escalationPolicy, service)
 }
 
+func testAccCheckPagerDutyServiceConfigWithAlertTimeGroupingUpdated(username, email, escalationPolicy, service string) string {
```

Comment:
This function should be moved up to line 1087, right after `testAccCheckPagerDutyServiceConfigWithAlertContentGroupingUpdated()`, to keep the same order as for invocations in test steps.

https://github.com/PagerDuty/terraform-provider-pagerduty/pull/582/files#diff-afdb38b0f8715bc060e1cca7ee93367e1fd6e8f41ac67a59d9c7f7660e003041R1086